docs/advanced.md

```diff
@@ -250,83 +250,76 @@ async with httpx.Client(proxies=proxy) as client:
     has not been implemented yet. To use proxies you must pass the proxy
     information at `Client` initialization.
 
-## Timeout fine-tuning
+## Timeout Configuration
 
-HTTPX offers various request timeout management options. Three types of timeouts
-are available: **connect** timeouts, **write** timeouts and **read** timeouts.
+HTTPX is careful to enforce timeouts everywhere by default.
 
-* The **connect timeout** specifies the maximum amount of time to wait until
-a connection to the requested host is established. If HTTPX is unable to connect
-within this time frame, a `ConnectTimeout` exception is raised.
-* The **write timeout** specifies the maximum duration to wait for a chunk of
-data to be sent (for example, a chunk of the request body). If HTTPX is unable
-to send data within this time frame, a `WriteTimeout` exception is raised.
-* The **read timeout** specifies the maximum duration to wait for a chunk of
-data to be received (for example, a chunk of the response body). If HTTPX is
-unable to receive data within this time frame, a `ReadTimeout` exception is raised.
+The default behavior is to raise a `RequestException` after 5 seconds of
```

Comment:
Isn’t it a `RequestTimeout`?

Reply:
Thanks!

What do we *actually* want the base timeout exception to be called? `TimeoutError` is a python built-in, so I guess we've got...

* `TimeoutException`
* `BaseTimeout`
* `RequestTimeout`

?...

Reply:
I like `TimeoutException` best: explicit, boring and unsurprising. (`RequestTimeout` has an odd vibe to it that I can't really explain, and `BaseTimeout` sounds like a base class.)